### ISSUE TITLE
Switch from HTTP 302 to HTTP 303

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SeasideSt/Seaside#782]
```
http://insanecoding.blogspot.ch/2014/02/http-308-incompetence-expected.html
```

Original issue reported on code.google.com by `philippe...@gmail.com` on 18 Feb 2014 at 9:15

**Comment by jbrichau** [comment]
In the context of Hotwire Turbo, this is now required to return redirect 303 to trigger certain behaviors. See also: https://github.com/hotwired/turbo/issues/84

I think Seaside should redirect using 303 in all cases.

**Comment by marschall** [comment]
Yes 303 specifies the GET after POST behavior we want https://en.wikipedia.org/wiki/HTTP_303
